compiler/compiler.go

```diff
@@ -198,22 +198,29 @@ func (c *Compiler) Compile(mainPath string) []error {
 			Compiler:    "gc", // must be one of the recognized compilers
 			BuildTags:   append([]string{"tinygo", "gc." + c.selectGC()}, c.BuildTags...),
 		},
-		ShouldOverlay: func(path string) bool {
+		OverlayPath: func(path string) string {
+			// Return the (overlay) import path when it should be overlaid, and
+			// "" if it should not.
+			if strings.HasPrefix(path, "github.com/tinygo-org/tinygo/src/") {
```

Comment:
This should probably be a separate `const` for when we switch over the main repo to `tinygo.org/x/tinygo`.

Reply:
Fixed. Yes, I agree that's better.